products/freebsd.md

```diff
@@ -6,27 +6,31 @@ releasePolicyLink: https://www.freebsd.org/security/#sup
 activeSupportColumn: false
 releaseDateColumn: false
 releaseColumn: false
-versionCommand: freebsd-version
+versionCommand: freebsd-version -kru
 releases:
 -   releaseCycle: "releng/13.1"
     eol: false
     # 13.2-RELEASE + 3 months
-    link: https://www.freebsd.org/releases/13.1R/announce/
```

Comment:
In the **Release** column at <https://endoflife.date/freebsd>, the hint when pointing at a link is: 

> Release Notes / Changelog for …

`/announce/` URLs are for announcements – **not** release notes. 

The shorter form of URL allows the reader to more easily see the **direct link** to release notes, alongside links to other pages that are relevant to the release.

Reply:
https://www.freebsd.org/releases/13.1R/ -> https://www.freebsd.org/releases/13.1R/relnotes/
same for others

Reply:
> [freebsd.org/releases/13.1R](https://www.freebsd.org/releases/13.1R/) -> [freebsd.org/releases/13.1R/relnotes](https://www.freebsd.org/releases/13.1R/relnotes/) 

Usually, a direct link to release notes would be a first choice. 

Unusually, FreeBSD release notes **lack the date of release** (and I do expect some interest in dates in the `endoflife.date` site context). 

The shorter form of URL allows the reader to see, _at a glance_: 

* the link to release notes
* the link to the schedule, which includes the release date

– and so on.

Reply:
I agree with @grahamperrin for this particular case.

Reply:
Thanks. 

On the FreeBSD side: https://reviews.freebsd.org/D37528 aims for future improvements.

Reply:
Then lets keep it like @grahamperrin offers, but we still need a proper changelogtemplate to overcome having lots of link tags @marcwrobel

Reply:
`changelogTemplate` added.

Link for stable release must be set manually : I do not see a way to build a `changelogTemplate` compatible with those cycles.